Makefile

```diff
@@ -44,7 +44,7 @@ containers-storage: $(sources) ## build using gc on the host
 	$(GO) build $(MOD_VENDOR) -compiler gc $(BUILDFLAGS) ./cmd/containers-storage
 
 codespell:
-	codespell -S Makefile,build,buildah,buildah.spec,imgtype,copy,AUTHORS,bin,vendor,.git,go.sum,CHANGELOG.md,changelog.txt,seccomp.json,.cirrus.yml,"*.xz,*.gz,*.tar,*.tgz,*ico,*.png,*.1,*.5,*.orig,*.rej" -L flate,uint,iff,od,ERRO -w
+	codespell -S Makefile,build,buildah,buildah.spec,imgtype,copy,AUTHORS,bin,vendor,.git,go.sum,CHANGELOG.md,changelog.txt,seccomp.json,.cirrus.yml,"*.xz,*.gz,*.tar,*.tgz,*ico,*.png,*.1,*.5,*.orig,*.rej" -L worl,flate,uint,iff,od,ERRO -w
```

Comment:
What's the reasoning behind adding "worl" to the list of words it ignores?

Reply:
$ grep '" worl' pkg/ioutils/buffer_test.go
	if string(b) != " worl" {
		t.Fatalf("expected \" worl\", got %s", string(b))

Reply:
It wants to change this test, which then breaks.

Reply:
Ah, codespell 2.2, not 2.1 that we have in Fedora 37.